Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -160,13 +169,15 @@ def StateActivity(self, state, mouse1, mouse2):
             mouse1.DeliverReward(Locations.Defect, self.punishment_time)
             mouse2.DeliverReward(Locations.Defect, self.punishment_time)
             self.time_to_make_decision = time.time() - self.time_start
+            self.numcompletedtrial += 1
```

Comment:
rather than do it four time in each of the decision states you could have incremented the trial num and reset the return time  once in the trialCompleted state. Right?